src/executor.rs

```diff
@@ -400,6 +447,7 @@ pub enum StageType {
 pub enum TestStatus {
     Passed = 1,
     Failed = 2,
+    Skipped = 3,
```

Comment:
Will this be a problem?

Reply:
```
pub enum TestStatus {
    #[default]
    Unknown = 0,
    Pass = 1,
    Fail = 2,
    Pending = 3,
    Executing = 4,
}
```
This is what we have for the enumeration in the API, so I think Skipped would need to be 5.

Reply:
thanks